### PR TITLE
Add drawer navigation

### PR DESCRIPTION
**Pull request by psakurai**

### CODE REVIEW

**Review by mangkoran — CHANGES_REQUESTED** [review]
For icons please use @expo/vector-icons/Ionicons

File: screens/SupportUs.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const SupportUs = () => {
+return (
+	<View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+	  <Text>Support Us</Text>
+	</View>
+);
+};
+
+export default SupportUs;
```
Comment (mangkoran) [inline screens/SupportUs.js:12]: Create separate commit

**Review by mangkoran — CHANGES_REQUESTED** [review]
Please provide screenshot

**Comment by psakurai** [comment]
tab1
tab2

**Comment by mangkoran** [comment]
Refer to the latest design on penpot. We won't use drawer for settings. We will use hamburger menu instead.

**Comment by psakurai** [comment]
This PR will be closed due to unsupported modules.

**Pull request closed without merging**